src/PersonalNumber/UK.elm

```diff
@@ -0,0 +1,346 @@
+module PersonalNumber.UK exposing
+    ( NationalInsuranceNumber
+    , fromString, toString, display
+    , ValidationError(..), errorToString, simpleError
+    , decoder, encode
+    )
+
+{-| Parses UK National Insurance Numbers.
+
+With apologies to the government website for using an example that they asked me NOT to use,
```

Comment:
You should probably refrain from using an informal/personal voice in the documentation. Keep it short, objective and to the point.